deploy/Dockerfile

```diff
@@ -34,54 +29,59 @@ RUN apt update --fix-missing && apt install \
 # Set UTF-8 locale. Needed for the Rust compiler to handle Unicode column names.
 RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
   locale-gen
+
```

Comment:
I thought you were removing this file

Reply:
This one stays, but one in the enterprise repo can go away now